### DISCUSSION TITLE
Default Content layout (Table, Board,...) for item list

### DISCUSSION THREAD

**Discussion by ansutung** [directus/directus#21711] [Feature Requests]
### Summary

Currently, when an admin creates a new collection and multiple fields for that collection, the default layout displayed to users is a table layout with only a few fields visible. Users will not know which additional fields can be added without specific instructions. If admins could organize and arrange the necessary columns in a layout view and set it as the default, all users would have a better viewing experience with the information presented in a logical order.
Thanks

### Basic Example

...

### Motivation

...

### Detailed Design

...

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

...

### Alternatives

...

### Adoption Strategy

...

**Comment by JoshTheDerf** [comment]
This is doable with global bookmarks, but it isn't easily discoverable how to go about adding them.

**Reply by VladyslavChernyshov** [reply]
> This is doable with global bookmarks, but it isn't easily discoverable how to go about adding them.

Thank you, that helped, but still it would be more comfortable to have some sort of default bookmark for the whole data model.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
